packages/blockchain-wallet-v4-frontend/src/providers/ModalEnhancer/index.js

```diff
@@ -29,14 +29,14 @@ export default (type) => (Component) => enhance(
 
     handleClick (e) {
       const modalContainer = ReactDOM.findDOMNode(this.node)
-      if (modalContainer && equals(modalContainer.children[0], e.target)) {
+      if (modalContainer && equals(modalContainer.children[0], e.target) && !this.props.blockOnOutsideClick) {
         this.props.close()
       }
     }
 
     onKeyPressed (evt) {
       const event = evt || window.event
-      if (event.keyCode === 27) {
+      if (event.keyCode === 27 && !this.props.blockEscKeypress) {
```

Comment:
it allows for modals to not close when user presses the escape key.  since the error screen is a modal, we need to force them to click the continue button to redirect them to either homepage or login page.  if we allow them to escape out of the modal then they will be sitting on a broken white page.